kratos/input_output/logger_message.h

```diff
@@ -74,16 +75,75 @@ namespace Kratos
 				CHECKING
 			};
 
+			class DistributedFilter {
+			public:
+				DistributedFilter()
+					: mIsDistributed(false), mPrintFromAllRanks(false), mSourceRank(0) {}
+
+				DistributedFilter(bool IsDistributed, bool PrintFromAllRanks, int TheRank)
```

Comment:
This one can be private. Isn't it?

Reply:
And I would not get the FromAllRanks flag if you are asking for TheRank

Reply:
Is there any specific reason to not use local flags here?

Reply:
Not really, it is just that I was hoping to be able to work with a single bool (but that would require a separate MPI LoggerOutput, I believe). I can make it work with local flags if we decide to go with the current design.

Reply:
I don't have an strong recommendation on this one as they are only two booleans. As it will be used internally we have time to change it.